src/bin/common/parse_command.ml

```diff
@@ -936,8 +976,32 @@ let parse_output_opt =
     Arg.(value & opt (some format_conv) None & info ["o"; "output"] ~docv ~doc)
   in
 
+  let model_type =
+    let doc =
+      Format.sprintf
+        "Control the output model type of the solver, $(docv) must be %s."
+        (Arg.doc_alts [ "value"; "constraint" ])
+    in
+    let docv = "MTYP" in
+    Arg.(
+      value &
+      opt (some model_type_conv) None &
+      info ["mt"; "model-type"] ~docv ~doc
+    )
+  in
+
+  let mdls =
+    let doc =
+      "Simply activates the models in alt-ergo. This is achieved by setting \
+       some parameters by default: interpretation = every; instanciation \
+       heuristic = normal; \
+       sat-solver = tableaux"
+    in
+    Arg.(value & flag & info ~doc ~docs ["models"])
```

Comment:
I think the default should be to provide one model, and use the `before_end` setting. Outputting more than one model by default will confuse users. And so, also remove the `s` to the option name.